codegen/codegen-core/src/main/java/software/amazon/smithy/java/codegen/CodegenUtils.java

```diff
@@ -255,6 +255,19 @@ public static String toDefaultValueName(String memberName) {
         return CaseUtils.toSnakeCase(memberName).toUpperCase(Locale.ENGLISH) + "_DEFAULT";
     }
 
+    /**
+     * Gets the name to use when defining the default value of a member.
+     *
+     * @param memberShape memberShape.
+     * @return Upper snake case name of default
+     */
+    public static String toGetterName(MemberShape memberShape, String memberName) {
```

Comment:
We should probably use `CodegenUtils.MEMBER_ESCAPER.escape` here like the old code did. If you pass the resolved name of getX to the escaper, it'll ensure we don't clash with anything like getClass.

Reply:
I created a new method escaper since there is no overlap with member escaper.